db/migrate/20240314225751_create_usertocourses.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsertocourses < ActiveRecord::Migration[7.1]
+  def change
+    create_table :usertocourses do |t|
```

Comment:
```suggestion
    create_table :user_to_courses do |t|
```
SQL convention is to snake_case multi-word table names.